forte/data/data_pack.py

```diff
@@ -867,25 +868,54 @@ def get_data(
         valid_context_ids: Set[int] = self._index.query_by_type_subtype(
             context_type_
         )
+
         if context_components:
             valid_component_id: Set[int] = set()
             for component in context_components:
                 valid_component_id |= self.get_ids_by_creator(component)
             valid_context_ids &= valid_component_id
 
+        def get_annotation_list(c_type):
+            """
+            Get an annotation list of a given context type.
+            """
+            if issubclass(c_type, Annotation):
+                return list(self.annotations)
+            elif issubclass(c_type, AudioAnnotation):
+                return list(self.audio_annotations)
+            else:
+                raise NotImplementedError(
+                    f"Context type is set to {c_type}"
+                    "but currently we only support"
+                    " Annotation, AudioAnnotation."
+                )
+
+        def get_context_data(c_type, context):
+            """
+            Get context data of a given context type and context.
+            """
+            if issubclass(c_type, Annotation):
+                return self.text[context.begin : context.end]
+            elif issubclass(c_type, AudioAnnotation):
+                return self.audio[context.begin : context.end]
+            else:
+                raise NotImplementedError(
+                    f"Context type is set to {context_type}"
+                    "but currently we only support"
+                    "{Annotation, AudioAnnotation}"
```

Comment:
This is similar to above, let's resolve this too. You are using `{` here, which is near an f-string construct. This could be confusing to future developers. They might think you forget to add a f before this line. Let's not use `{` here, maybe use `[`

Reply:
Action required

Reply:
Fixed it.